lib/components/PlaylistPage.js

```diff
@@ -1,15 +1,78 @@
 import React from 'react';
+import Request from 'superagent';
+
 import SearchBar from './SearchBar';
+import SearchResult from './SearchResult';
+import SearchResults from './SearchResults';
 
 const INPUT_KLASS = "spotify-search"
 const INPUT_PLACEHOLDER = "Search Spotify"
 
 export default React.createClass({
+  getInitialState() {
+    return {
+      playlistTracks : {},
+      searchResults : [],
+    }
+  },
+
+  handleSearch(e) {
+    e.preventDefault();
+    let queryValue = e.target.value;
+    this.searchSpotify(queryValue);
+  },
+
+  storeResults(tracks) {
+    debugger;
+    this.state.searchResults = tracks;
+    this.setState({ searchResults : self.state.searchResults });
+  },
```

Comment:
yup, what Allan said! 

Let me know if that doesn't solve your problem @scottfirestone and I can take a closer look.